.travis.yml

```diff
@@ -1,8 +1,7 @@
 os: linux
-dist: trusty
 language: python
 # 'sudo' is enabled automatically by the 'apt' addon below.
-#sudo: false
+sudo: required
```

Comment:
```suggestion
```
This can now be removed: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration